js/feathergrid.ts

```diff
@@ -885,6 +871,15 @@ export class FeatherGrid extends Widget {
     });
   }
 
+  _isSelectionSelected(): boolean {
+    if (this.grid.selectionModel && !this.grid.selectionModel.isEmpty) {
+      return true;
+    } else {
+      return false;
+    }
+    // return this.grid.selectionModel && this.grid.selectionModel.isEmpty;
+  }
+
```

Comment:
I'd remove this and just use the expression in the `isEnabled` option

```suggestion
```

Reply:
`isEnabled` requires a true/false value, and `this.grid.selectionModel` can potentially be `null` , I think it should work if I use `!!this.grid.selectionModel && !this.grid.selectionModel.isEmpty` if that's preferable.

Reply:
or `this.grid.selectionModel == null || this.grid.selectionModel.isEmpty`